build.gradle

```diff
@@ -641,9 +642,9 @@ project('spring-integration-mail') {
 		api project(':spring-integration-core')
 		api 'org.springframework:spring-context-support'
 
-		providedImplementation "jakarta.mail:jakarta.mail-api:$mailVersion"
-		providedImplementation "com.sun.mail:imap:$mailVersion"
 		providedImplementation "com.sun.mail:jakarta.mail:$mailVersion"
+		testImplementation "com.icegreen:greenmail:$greenmailVersion"
+		testImplementation "org.slf4j:jul-to-slf4j:1.7.30"
```

Comment:
Why do we need this extra dependency?
Nevertheless it is better to wrap literal into a single quotes in Groovy: otherwise it is treated as GString and evaluated when Gradle build is executed. Kinda some performance optimization.